### ISSUE TITLE
Delete multiple Chips

### ISSUE THREAD

**Issue by davin12x** [fiskurgit/ChipCloud#8]
After deleting first chip . How can i refresh chips . Next time  i select first chip again it gives me position 1 instead of zero

**Comment by fiskurgit** [comment]
You've found a bug, I'll have a fix out some time this week.

**Comment by codekidX** [comment]
how do you delete a chip from chipCloud ? I can't find it any method for it

**Comment by fiskurgit** [comment]
Sorry, I've not had time to look into this yet. The library wasn't intended to handle this use-case, but would be good to have it.

**Comment by davin12x** [comment]
Use this property.
chipCloud.removeViewAt(indexPosition);

**Comment by codekidX** [comment]
@fiskurgit It's all right I thought I missed it. I'll fork and add it and submit pull request or you can cherry-pick.

@davin12x  I tried that but my app crashes and android studio doesn't catch it in its log too

**Comment by fiskurgit** [comment]
Sorry this was left for so long. The bug was because the chips hold their index internally, so when you delete one in the middle of the array they become out of sync, solution was to reinitialise the indexes when one is removed. 

Working code to delete a selected chips as example:

```
final ChipCloud chipCloud = (ChipCloud) findViewById(R.id.chip_cloud_a);
String[] demoLabels = new String[]{"UY Scuti", "WOH G64", "RW Cephei", "Westerlund 1-26", "V354 Cephei", "KY Cygni", "VY Canis Majoris"};
chipCloud.addChips(demoLabels);

Button deleteButton = (Button) findViewById(R.id.delete_selected_button);
deleteButton.setOnClickListener(new View.OnClickListener() {
  @Override public void onClick(View view) {
    int selectedIndex = chipCloud.getSelectedIndex();
    chipCloud.deleteChip(selectedIndex);
  }
});
```

getSelectedIndex() and deleteChip(int index) methods available in 2.2.1

**Comment by CyberpunkPsychonaut** [comment]
deleteChip is no longer an available method, don't see any way to delete a chip(s)
